templates/vim-alphabet.text

```diff
@@ -0,0 +1,17 @@
+Learning Goal: Learning the basics of the Vim Text Editor from A to Z
+
+Learning Curriculum: youtube.com/playlist?list=PLnc_NxpmOxaNqdGvUg8RBi8ZTaZGPdqBD
+
+Special Requirements:
+- {Familiarity with the Terminal and Text Editors}
+
+
+
+Schedule:
+Meetings will be async. Suggested watch and learn two letters a day and discuss them through the group.
+
+Meetings are Sync: {No}
+
+Club Captain: {captain's name} ({captain's discord username})
```

Comment:
The braces here *are* necessary though so it's more clear that these need to be filled in by whoever's using this template.